frontend/tests/e2e-tests.sh

```
@@ -13,67 +13,65 @@ MAILER_WEB_SERVER_PORT=8073
+  elif [[ $arg == -k ]]; then
+    KEEP_DATABASE_SNAPSHOT=1
+  elif [[ $arg == --* ]]; then
+    SCRIPT_LONG_ARGS+=("$arg")
+  elif [[ $arg == -* ]]; then
+    SCRIPT_SHORT_ARGS+=("$arg")
+  elif [[ $arg != -* ]]; then
+    TEST_PATHS+=("$arg")
+  fi
 done
 
 if [[ " ${SCRIPT_SHORT_ARGS[@]} " =~ " -h " ]] || [[ " ${SCRIPT_LONG_ARGS[@]} " =~ " --help " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)